scripts/buildRedirectConfig.js

```diff
@@ -10,7 +10,8 @@ const legacyRedirects = {
     "^/Pages/AdvancedSearch.aspx/": "/search",
     "^/DownloadCenter/Pages/DataDownload.aspx/": "/download_center/custom_award_data",
     "^/DownloadCenter/Pages/dataarchives.aspx/": "/download_center/award_data_archive",
-    "^/covid-19/": "/disaster/covid-19"
+    "^/covid-19/": "/disaster/covid-19",
+    "^/analyst-guide/": "/federal-spending"
```

Comment:
should this be 'federal-spending-guide'?

and how can we test the redirect locally?
i'm getting a 404 for localhost/analyst-guide, even after making the change suggested above, which i'm not certain is necessary

Reply:
OH it should be good catch!
 and I guess you can't locally only in qat i believe thats what @anjenkin said.